packages/aws-cdk-lib/aws-apigatewayv2-integrations/lib/http/sqs.ts

```diff
@@ -0,0 +1,117 @@
+import * as apigwv2 from '../../../aws-apigatewayv2';
+import * as iam from '../../../aws-iam';
+import * as sqs from '../../../aws-sqs';
+
+/**
+ * Properties to initialize `HttpSqsIntegration`.
+ */
+export interface HttpSqsIntegrationProps {
+  /**
+   * Specifies how to transform HTTP requests before sending them to the backend.
+   *
+   * When the subtype is either `START_EXECUTION` or `START_SYNC_EXECUTION`,
+   * it is necessary to specify the `StateMachineArn`.
+   * Conversely, when the subtype is `STOP_EXECUTION`, the `ExecutionArn` must be specified.
+   *
+   * @see https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-parameter-mapping.html
+   *
+   * @default - specify only `StateMachineArn`
+   */
+  readonly parameterMapping?: apigwv2.ParameterMapping;
```

Comment:
Are these parameter mappings typically defined or can they be whatever the user adds here?

Reply:
Default parameter mappings are based on this [docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services-reference.html#SQS-SendMessage).
I defined only required parameter such as `QueueUrl` for `SQS_SEND_MESSAGE`.

If a user wants to set optional parameters or change the values of required parameters, it is assumed that they will provide a custom-defined `parameterMapping`.

Is this response addressing your question? Please feel free to correct me if my understanding is incorrect.